lua/tests/automated/pickers/scrolling_spec.lua

```diff
@@ -1,12 +0,0 @@
-require("plenary.reload").reload_module "telescope"
-
-local tester = require "telescope.pickers._test"
-
-local log = require "telescope.log"
-log.use_console = false
-
-describe("scrolling strategies", function()
-  it("should handle cycling for full list", function()
-    tester.run_file [[find_files__scrolling_descending_cycle]]
-  end)
-end)
```

Comment:
uff, this test was still using the old testing framework.

can we fix it rather than deleting it?

I think it should be enough to replace the require call (we can also drop the reload und log stuff

```
local tester = require "telescope.testharness"
```

Reply:
ah nvm you moved it to `find_files_spec.lua`